docs/src/wallet-guide/apps.md

```diff
@@ -7,6 +7,30 @@ experience for most people who are new or experienced with using crypto wallets.
 currently the easiest and fastest way to get set up with a new wallet on Solana.
 The app is free and getting your wallet set up only takes a few minutes.
 
+### Trust Wallet Security
+
+Tokens held in Trust Wallet are only as secure as the device on which the app is
+installed.  Anyone who is able to unlock your phone or tablet may be able to
+use the Trust Wallet app and transfer your tokens.  It is *highly* recommended
+that you always use a strong passcode to lock your device and use a different
+passcode to open the app itself.  Enable Trust Wallet passcode by opening
+the app and going to Settings -> Security -> Passcode.
+
+Additionally, anyone who has access to your seed phrase will be able to recreate
```

Comment:
This is missing a step. It's more like anyone who gains access to the application can access your recovery phrase, and once someone has the recovery phrase, they can recreate your wallet on another device.